### PR TITLE
[DOCS] Improves influencer documentation

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR:
* improves the readability of the documentation on influencers by using a bulleted list of requirements
* put the requirements for the API users under a collapsible section
* suggests using the Advanced job wizard,
* adds further details about the topic.

### Preview

[Influencers](https://stack-docs_2221.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-ad-run-jobs.html#ml-ad-influencers)

### CODE REVIEW

File: docs/en/stack/ml/anomaly-detection/ml-influencers.asciidoc [hunk 1]
```diff
@@ -7,33 +7,44 @@ That is to say, _influencers_ are fields that you suspect contain information
...
-query results; since search hits cannot span multiple indices or documents,
-{dfeeds} have the same limitation. 
-
-Influencers do not need to be fields that are specified in your {anomaly-job}
-detectors, though they often are. If you use aggregations in your {dfeed}, it is
-possible to use influencers that come from different indices than the detector
-fields. However, both indices must have a date field with the same name, which you
-specify in the `data_description`.`time_field` property for the {dfeed}.
+[discrete]
+==== General information and requirements
```
[the remark this answers is not in the document]
Comment (chriscressman) [reply docs/en/stack/ml/anomaly-detection/ml-influencers.asciidoc:11]: > I  think @chriscressman once talked about hiding/calling out some content that should only be used by advanced users (or only after you've used the default/easy options with confidence)

Yes! I showed this README, which groups topics by basic and advanced:

https://github.com/elastic/enterprise-search-sharepoint-server-connector

And I also mentioned some JavaScript books I read that flag specific sections as advanced, which is a similar but different approach:

https://exploringjs.com/impatient-js/downloads/complete-toc.html

[the remark this answers is not in the document]
Comment (szabosteve) [reply docs/en/stack/ml/anomaly-detection/ml-influencers.asciidoc]: Thanks for your help to both of you.
I moved the item that is relevant for both the API users and the advanced wizard users to the main text. A new sentence is added to the text that suggests using the advanced job wizard. Finally, I used a collapsible section for the lower-level requirements that are important only for the API users. ([b01c066](https://github.com/elastic/stack-docs/pull/2221/commits/b01c0668a4770747523b8981a139c6ab95cfa8ae))

**Review by lcawl — APPROVED** [review]
This is a lot clearer! LGTM

File: docs/en/stack/ml/anomaly-detection/ml-influencers.asciidoc [hunk 2]
```diff
@@ -5,35 +5,49 @@ suspicions about which entities in your data set are likely causing
...
-detectors, though they often are. If you use aggregations in your {dfeed}, it is
-possible to use influencers that come from different indices than the detector
-fields. However, both indices must have a date field with the same name, which you
-specify in the `data_description`.`time_field` property for the {dfeed}.
+data. Influencers can be any field in your data.
+
+You can pick influencers when you create your {anomaly-job} by using the 
+**Advanced job wizard**.
+
+.Requirements when using the {ml} APIs to pick influencers
```
Comment (szabosteve) [inline docs/en/stack/ml/anomaly-detection/ml-influencers.asciidoc:13]:
```suggestion
.**Requirements when using the {ml} APIs to pick influencers**
```

**Comment by szabosteve** [comment]
@elasticmachine update branch

**Pull request merged**